app/main.py

```diff
@@ -1,2 +1,72 @@
+from typing import Any
+
+INC_SIZE_TWICE = 2
+
+
 class Dictionary:
-    pass
+    size = 8
```

Comment:
contants better with capital letters:
```suggestion
    SIZE = 8
```